### PR TITLE
Add mappings for SignEditScreen

### PR DESCRIPTION
**Pull request by enbrain**

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/ingame/SignEditScreen.mapping [hunk 1]
```diff
@@ -1,6 +1,10 @@
 CLASS net/minecraft/class_498 net/minecraft/client/gui/screen/ingame/SignEditScreen
+	FIELD field_21525 model Lnet/minecraft/class_837$class_4702;
+	FIELD field_24285 text [Ljava/lang/String;
```
Comment (YanisBft) [inline mappings/net/minecraft/client/gui/screen/ingame/SignEditScreen.mapping:3]:
```suggestion
	FIELD field_24285 textLines [Ljava/lang/String;
```

Comment (enbrain) [reply mappings/net/minecraft/client/gui/screen/ingame/SignEditScreen.mapping:3]: Should `SignBlockEntity.text` also be `textLines` then?

https://github.com/FabricMC/yarn/blob/5642461bdeb9cdd118718bca7101cb891c33ccf5/mappings/net/minecraft/block/entity/SignBlockEntity.mapping#L5

Comment (YanisBft) [reply mappings/net/minecraft/client/gui/screen/ingame/SignEditScreen.mapping:3]: Yeah probably

**Review by liach — APPROVED** [review]
Think `text` is good enough. It's just that text field which is made up of 4 lines.

**Review by YanisBft — APPROVED** [review]

**Review by Gegy — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

**Pull request merged**